tensorflow/lite/g3doc/performance/model_optimization.md

```diff
@@ -125,6 +125,37 @@ the numbers here:
   </figcaption>
 </figure>
 
+### Full integer quantization with int16 activations and int8 weights
+
+[Quantization with int16 activations](https://www.tensorflow.org/model_optimization/guide/quantization/post_training) is a full integer quantization scheme with activations in int16 and weights in int8. This mode can improve accuracy of the quantized model in comparison to the full integer quantization scheme with both activations and weights in int8 keeping a similar model size. It is recommended when activations are sensitive to the quantization.
```

Comment:
Could you please mention the expected performance comparison between the 16x8 quantization and the 8x8 quantization? This should give the reader more information for them to make the decision. Thanks.

Reply:
Hi @jianlijianli I added explanation regarding performance. Could you please take a look and re-approve if it is ok? Thanks